### ISSUE TITLE
Changing the function of the f12 key

### ISSUE THREAD

**Issue by akshay-49** [seerge/g-helper#440]
The Airplane Mode key on f12 is the most useless one, according to many people and me, as turning on airplane mode is something you do rarely.

It would be good if we could change the function of it to something else, like with fn+f4/m4/m3

**Comment by bel57** [comment]
Hey there, in case it doesn't get implemented- you can easily reassigns keybinds with PowerToys (official tool from Microsoft)
Check out keyboard manager, I assigned F12 to Win+Alt+G to easily take clips with Game Bar from a single key
![image](https://github.com/seerge/g-helper/assets/10614811/210b9c33-4007-49bb-b11b-8b9bbb6084c4)

**Comment by seerge** [comment]
@bel57 i agree, i'm not a big fan of adding things that you can do with Sharpkeys or Powetoys to the app. M1-M4 keys are there for a reason, you can't bind them otherwise.
